packages/rest-client-jsonapi/src/services/services.test.ts

```diff
@@ -104,73 +118,235 @@ describe("Testing CRUD operations against Hatchify backend", async () => {
+          database: process.env.PG_DB_NAME,
+          logging: false,
+        },
+      },
+    )
+    app.use(cors())
+    app.use(hatchedKoa.middleware.allModels.all)
+    await hatchedKoa.createDatabase()
+    const server = app.listen(3012) // TODO determine why unique port numbers are needed across tests in this file (otherwise, tests are liable to fail): https://bitovi.atlassian.net/browse/HATCH-415
+    const jsonApi = jsonapi(
+      `http://localhost:3012/${testBackendEndpointConfig.api}`,
+      {
+        Feature_Article: {
+          ...Feature_Article,
+          endpoint: `${testBackendEndpointConfig.namespacedSchemaSegment}`,
+        },
+        Admin_User: {
+          ...Admin_User,
+          endpoint: `${testBackendEndpointConfig.adminUserNamespaceSegment}`,
+        },
+      },
+    )
+    const hatchedReactRest = hatchifyReactRest(jsonApi)
+
+    await hatchedReactRest.Admin_User.createOne({
+      attributes: {
+        name: "Juno",
+      },
+    })
+
+    await hatchedReactRest.Feature_Article.createOne({
+      attributes: {
+        author: "John Doe",
+        tag: "Hatchify",
+      },
+    })
+
+    const [adminUsers] = await hatchedReactRest.Admin_User.findAll({})
+    const [adminUser] = adminUsers
+    const { id: userId } = adminUser
+    expect(adminUsers.length === 1)
+
+    expect(adminUser).toEqual({
+      id: userId,
+      __schema: "Admin_User",
+      name: "Juno",
+    })
+
+    const [featureArticles] = await hatchedReactRest.Feature_Article.findAll({})
+    const [featureArticle] = featureArticles
+    const { id: articleId } = featureArticle
+    expect(featureArticles.length === 1)
+
+    expect(featureArticle).toEqual({
+      id: articleId,
+      __schema: "Feature_Article",
+      admin_UserId: null,
+      admin_usersId: null,
+      author: "John Doe",
+      tag: "Hatchify",
+    })
+
+    await hatchedReactRest.Admin_User.updateOne({
+      id: userId,
+      attributes: {
+        name: "Juno Updated",
+      },
+      relationships: {
+        articles: [
+          {
+            id: articleId,
+          },
+        ],
+      },
+    })
+    const updatedAdminUserQuery = await hatchedReactRest.Admin_User.findOne({
+      id: userId,
+      include: ["articles"],
+    })
+
+    expect(updatedAdminUserQuery).toEqual({
+      id: userId,
+      name: "Juno Updated",
+      __schema: "Admin_User",
+      articles: [
+        {
+          id: articleId,
+          __schema: "Feature_Article",
+          __label: "John Doe",
+          author: "John Doe",
+          tag: "Hatchify",
+          admin_usersId: null,
+          admin_UserId: userId,
+        },
+      ],
+    })
+
+    await hatchedReactRest.Feature_Article.updateOne({
+      id: articleId,
+      attributes: {
+        author: "John Doe Updated",
+        tag: "Hatchify Updated",
+      },
+      relationships: {
+        admin_users: {
+          id: userId,
+        },
+      },
+    })
+
```

Comment:
can we also add a test that does the opposite, ie. `Admin_User.updateOne(.... relationships: [])`, just to make sure the updateOne works both when called against a to-one relationship (object w/ id) and a to-many relationship (array of objects w/ id)